docs/sql/statements/create-repository.rst

```diff
@@ -669,6 +669,52 @@ Parameters
 
   The container path to use for snapshots.
 
+.. _sql-create-repo-gcs-compress:
+
+**compress**
+  | *Type:*    ``boolean``
+  | *Default:* ``true``
+
+  Whether CrateDB should compress the metadata part of the snapshot or not.
+
+  CrateDB does not compress the actual table data.
```

Comment:
```suggestion
```

I think this may be read as if CrateDB doesn't do compression at all, which isn't the case.
I'd tend to remove it, otherwise we should probably clarify that it doesn't apply additional compression on top of the already compressed data.